nucypher/config/node.py

```diff
@@ -429,9 +435,15 @@ def dynamic_payload(self, **overrides) -> dict:
                        known_nodes=self.known_nodes,
                        node_storage=self.node_storage,
                        crypto_power_ups=self.derive_node_power_ups() or None)
+
+        if self.federated_only is False:
```

Comment:
Probably `if not self.federated_only`

Reply:
With consideration to `None`?

Reply:
You have it hinted as a `bool`, so I suppose this is actually fine.

My thinking was simply that, since this is a public class, people can pass any object they like (so long as it implements `__bool__`) to its `__init__` and, ideally, we're supposed to accept it.  How do we know that someone doesn't have a higher-order state machine for determining `federated_only`?

But yeah, I actually think this is a non-issue.

Reply:
Yeah, I think we have several unnecessary `is False` / `is True`

Reply:
:heavy_check_mark: